src/main/resources/project/ui_forms/deploy_DE.xml

```diff
@@ -87,6 +87,16 @@
         <dependsOn>connection_config</dependsOn>
     </formElement>
     <formElement>
+        <label>Region:</label>
+        <property>region</property>
+        <value/>
+        <documentation>The region zone in which to launch the server.</documentation>
+        <required>0</required>
+        <type>entry</type>
+        <serverOptions>0</serverOptions>
+        <dependsOn>connection_config</dependsOn>
+    </formElement>
+    <formElement>
```

Comment:
Between availability zone and region, is at least one of them required? If so, we should add it to the validations for dynamic envts. 
We also need to add the drop-down for regions and have it be populated dynamically similar to availability zones.